### DISCUSSION TITLE
Exterior/Physics/2/11/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#9] [General]
# Exterior/Physics/2/11/

https://savia7582.github.io/Exterior/Physics/2/11/

**Comment by ffy6511** [comment]
这一页的页数好像少了？这里只有一页😮

**Comment by cnxkxkxk** [comment]
这里是不是少了

**Comment by Fywt1** [comment]
电流和磁场这章中纯电流内容就这么多吧：）
